pyresample/gradient/__init__.py

```diff
@@ -402,3 +402,70 @@ def _concatenate_chunks(chunks):
     res = da.concatenate(res, axis=2).squeeze()
 
     return res
+
+
+class GradientSearchResampler(BaseResampler):
+    """Resample using gradient search based bilinear interpolation."""
```

Comment:
How about the nearest neighbour variant?

Reply:
yes, needs to be added.

Reply:
Done